AirCasting/CoreData/SessionEntity.swift

```diff
@@ -132,3 +132,9 @@ extension SessionEntity {
     @NSManaged public func removeFromMeasurementStreams(_ values: NSSet)
 
 }
+
+extension SessionEntity {
```

Comment:
wait, we already have such extension with different session types

Reply:
![Screenshot 2021-08-20 at 14 48 10](https://user-images.githubusercontent.com/46443841/130235488-311e844d-daf8-4b54-aeda-88bead981eb3.png)

Reply:
could you move isActive there?

Reply:
for sure, it is a bit mixed 😅